pilot/pkg/networking/core/v1alpha3/listener_test.go

```diff
@@ -509,6 +510,23 @@ func verifyInboundHTTPListenerCertDetails(t *testing.T, l *xdsapi.Listener) {
 	}
 }
 
+func verifyInboundHTTPListenerNormalizePath(t *testing.T, l *xdsapi.Listener) {
+	t.Helper()
+	if len(l.FilterChains) != 2 {
+		t.Fatalf("expected %d filter chains, found %d", 2, len(l.FilterChains))
```

Comment:
"expected 2 filter chains, found %d". 

Since 2 is a constant, we do not need to use %d.

Reply:
Fixed.